src/frontend/components/footer/Footer.jsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./Footer.css";
+
+
+export function Footer() {
+    return (
+        <footer className="ecom-footer flex">
+			<div className="footer-section flex-col gap-1rem">
+				<Link to="/" className="brandName">
+					Lappy Store
+				</Link>
+				<div className="footer-heading">
+					Made with ❤️ by Digambar
+				</div>
+				<div className="flex gap-1rem">
+					<a href="https://www.linkedin.com/in/digambar-deshawal-9b279b147/">
```

Comment:
Please add target blank attribute to anchor tag so that it will open in a new tab rather than opening in a same tab.

Reply:
thanks for pointing out it has been added now